src/dbs/index.ts

```diff
@@ -0,0 +1,12 @@
+import * as mongoose from 'mongoose';
+import { env } from '../env.config';
+
+(<any>mongoose).Promise = Promise;
+
+const connection : mongoose.Connection = mongoose
+    .createConnection(
+        env.MONGODB_CONNECTION, 
```

Comment:
It looks like in some files there are `4 spaces` indentation.
Are we going to change to 4, o should we keep in 2?